### PR TITLE
Feature/app 390 fix runtime errors

### PR DESCRIPTION
**Pull request by ingoCollatz**

### CODE REVIEW

File: shell/src/shared/openLogin.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export async function getOpenLogin(): Promise<GetOpenLoginResult> {
           privKey: acc.privateKey,
         };
       },
+      async logout(): Promise<void> { },
```
Comment (codeho) [inline shell/src/shared/openLogin.ts:18]: this is not doing anything... why did you add it? :)

File: shell/src/shared/processes/shellProcess.ts [hunk 2]
```diff
@@ -54,6 +54,7 @@ const processDefinition = () => {
           id: childProcessId,
           src: (context) => {
             const sm = context.childProcessDefinition.stateMachine(childProcessId);
+            console.log(`Running child process ${processId}: `, sm);
```
Comment (codeho) [inline shell/src/shared/processes/shellProcess.ts:57]: remove console.log

File: shell/src/main.ts [hunk 3]
```diff
@@ -124,7 +124,7 @@ window.o = <any>{
     },
     async run<TContext>(definition: ProcessDefinition<any, any>, contextModifier?: (processContext: ProcessContext<any>) => Promise<TContext>) {
       const processId = Generate.randomHexString(8);
-      console.log(`Starting process (id: ${processId}) with definition:`, definition);
+      console.log(`Starting process (id: ${processId}) with definition:`, definition, definition.stateMachine("debug"));
```
Comment (codeho) [inline shell/src/main.ts:127]: remove console.log

**Review by codeho — CHANGES_REQUESTED** [review]

**Review by codeho — APPROVED** [review]

**Review by jaensen — APPROVED** [review]
Looks good. Only one remark:
In one case you used 
```
import { get } from 'svelte/store'
```
and then 
```
      const $me = get(me)
```
in the other case you left
```
    let $me: Profile;
    me.subscribe((m) => ($me = m))();
```
which in principle is fine. But using the `get()` shorthand everywhere would surely have been nicer (I know, the latter was by me 😂).

**Pull request merged**